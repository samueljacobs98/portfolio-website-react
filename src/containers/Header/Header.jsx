import logo from "../../assets/svgs/logo.svg";
import Navbar from "../../components/Navbar/Navbar";
import "./Header.scss";

const Header = () => {
  return (
    <div className="header">
      <img className="logo__svg" src={logo} alt="Samuel Jacobs Logo" />
      <Navbar />
    </div>
  );
};

export default Header;
