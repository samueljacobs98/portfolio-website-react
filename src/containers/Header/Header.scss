@use "../../assets/scss/variables" as *;
.header {
  padding: $padding-header;
  display: flex;
  justify-content: space-between;
  align-items: center;
}
