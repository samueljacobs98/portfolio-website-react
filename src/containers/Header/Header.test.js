import { render } from "@testing-library/react";
import Header from "./Header";

test("should match snapshot", () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});
