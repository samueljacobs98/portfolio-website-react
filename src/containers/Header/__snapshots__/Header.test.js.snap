// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <div
    class="header"
  >
    <img
      alt="Samuel Jacobs Logo"
      class="logo__svg"
      src="logo.svg"
    />
    <ul
      class="navbar"
    >
      <li>
        <a
          class="navbar__link"
          href="#projects"
        >
          Projects
        </a>
      </li>
      <li>
        <a
          class="navbar__link"
          href="#about"
        >
          About
        </a>
      </li>
      <li>
        <a
          class="navbar__link"
          href="#contact"
        >
          Contact
        </a>
      </li>
    </ul>
  </div>
</div>
`;
